packages/flutter_tools/templates/plugin/android-java.tmpl/build.gradle.tmpl

```diff
@@ -2,13 +2,15 @@ group '{{androidIdentifier}}'
 version '1.0'
 
 buildscript {
+    ext.kotlin_version = '1.4.32'
```

Comment:
I understand the app change, but why does an Android *plugin* need this?

Reply:
This isn't needed. done